src/test/java/org/apache/cassandra/sidecar/routes/StreamSSTableComponentHandlerTest.java

```diff
@@ -315,9 +316,10 @@ void testSuffixRangeExceeds(VertxTestContext context)
                 .send(context.succeeding(response -> context.verify(() ->
                 {
                     assertThat(response.statusCode()).isEqualTo(OK.code());
+                    AbstractStringAssert<?> as =
                     assertThat(response.getHeader(HttpHeaderNames.CONTENT_LENGTH.toString()))
-                        .isEqualTo("4")
-                        .describedAs("Server should shrink the range to the file length");
+                        .describedAs("Server should shrink the range to the file length")
```

Comment:
`describeAs` is annotated with `org.assertj.core.util.CheckReturnValue` which is documented as `Findbugs handles any annotation with name "CheckReturnValue" in return value check.` . This was failing the build with warning `RV_RETURN_VALUE_IGNORED: Method ignores return value`